### DISCUSSION TITLE
Lolin D32 restarting without stopping

### DISCUSSION THREAD

**Discussion by ThiagoFernandes89** [micropython/micropython#13338] [ESP32]
Can someone help me?
I was able to observe that if you keep the reset button pressed it is possible to start recording and access the serial, but as soon as the reset is released, it restarts again without stopping.

**Comment by robert-hh** [comment]
Which firmware do you try to flash?
Where did you get it from?
Which flash tool do you use?

**Comment by ThiagoFernandes89** [comment]
Hello, sorry for the delay, I had some setbacks.
I've made some progress, but I'm still having problems.
I managed to record the firmware ESP32_GENERIC-OTA-20231227-v1.22.0.bin, using esptool, but I discovered that the problem with my cards is in the wifi, I'm using Tonny and I did the following test, whenever I try to activate the wifi it restarts .

import network
sta = network.WLAN(network.STA_IF)
sta.active(True)

**Reply by robert-hh** [reply]
That could be a power supply problem, either with the PC or the USB cable. Even if it is unlikely the reason, I would try the same without  Thonny, like using mpremote of Putty as terminal.

Just for interest: Why do you use the OTA version? Do you plan to update the firmware over-the-air?

**Reply by ThiagoFernandes89** [reply]
Initially I had thought about it, but I tested it on the PC, cell phone, battery and with other cables and I also use it with other ESP32s.

Isso mesmo, utilizar OTA.
